Assets/MRTK/SDK/Features/UX/Scripts/RiggedHandVisualizer/RiggedHandVisualizer.cs

```diff
@@ -165,104 +163,100 @@ private Quaternion UserBoneRotation
             }
         }
 
-        private Quaternion Reorientation()
-        {
-            return Quaternion.Inverse(Quaternion.LookRotation(ModelFingerPointing, -ModelPalmFacing));
-        }
-
-        private Dictionary<TrackedHandJoint, Transform> joints = new Dictionary<TrackedHandJoint, Transform>();
-        private Dictionary<TrackedHandJoint, Transform> skeletonJoints = new Dictionary<TrackedHandJoint, Transform>();
+        protected readonly Transform[] riggedVisualJointsArray = new Transform[ArticulatedHandPose.JointCount];
```

Comment:
Is there any conceptual overlap here between this and `jointsArray` in `BaseVisualizer` (and potential for reuse)? Or is there no reasonable way to reuse one array of joint transforms in this scenario?

Reply:
There's no reasonable way to reuse the array, since the base visualizer populates the array with an unstructured collection of joint transforms which represent the various skeleton joints.